Makefile

```
@@ -266,6 +266,15 @@ integration:
 	@echo KUBECONFIG is: $(KUBECONFIG), TEST_KUBE: $(TEST_KUBE)
 	go test -tags "$(PAM_TAG) $(FIPS_TAG) $(BPF_TAG)" ./integration/... $(FLAGS)
 
+#
+# Chaos tests. Run without race detector and have TestChaos prefix.
+#
+.PHONY: chaos
+chaos: FLAGS ?= -v -cover
+chaos: FOLDERS := $(shell find . -type f -name '*chaos*' | xargs dirname | uniq)
```

This pulls in matches from `vendor` and non-go directories.  Probably more future-proof to do something like this:

```suggestion
chaos: FOLDERS := $(shell find . -type f -name '*chaos*.go' -not -path "*/vendor/*" | xargs dirname | uniq)
```